webpack.config.js

```diff
@@ -122,6 +122,7 @@ const webpackConfig = {
 		],
 		alias: Object.assign(
 			{
+				'components/': '@automattic/simple-components/',
```

Comment:
This was due to (broken) local installation of lerna. Fixed by globally installing it instead.